ramls/oai-pmh.raml

```diff
@@ -0,0 +1,131 @@
+#%RAML 1.0
+title: OAI-PMH API
+baseUri: https://github.com/folio-org/mod-oai-pmh
+version: v1
+protocols: [ HTTP, HTTPS ]
+
+documentation:
+  - title: OAI-PMH Business Logic API
+    content: |
+      The Open Archives Initiative Protocol for Metadata Harvesting (OAI-PMH) provides
+      an application-independent interoperability framework based on metadata harvesting.
+      This module supports the OAI-PMH as a means of exposing FOLIO metadata.
+
+types:
+  oai-pmh: !include schemas/OAI-PMH.xsd
+
+traits:
+  partionable: !include traits/partitionable.raml
+  selective: !include traits/selective.raml
+  metadataPrefix: !include traits/metadataPrefix.raml
+
+resourceTypes:
+  list: !include rtypes/list.raml
+
+/oai:
+  /records:
+    displayName: Records
+    type:
+      list:
+        exampleCollection: !include examples/records.sample
+    get:
+      is: [ partionable, selective, metadataPrefix ]
+      responses:
+        404:
+          description: "Item with a given ID and selective harvesting params (if any) not found"
+          body:
+            application/xml:
```

Comment:
According to [OAI-PMH Spec 3.1.2.1 Content-Type](http://www.openarchives.org/OAI/openarchivesprotocol.html#MIMETypes)
> The Content-Type returned for all OAI-PMH requests must be text/xml.

So probably it makes sense to use `text/xml` instead of  `application/xml`

Reply:
This note is critical for EDGE module API.
This back-end module does not strictly follows OAI-PMH Spec anyway by exposing more RESTfull version of the API. Since there is no real difference between these two types (and sometimes it is even more preferable to use application/xml) I would leave it as is.

Reply:
I agree with @epam-pavelk here, but good to point out @piotr-kalashuk